10_reduced-precision/PyTorch/README.md

```diff
@@ -33,5 +36,34 @@ Essentially, the only difference is: instead of wrapping the optimizer object wi
 scaling optimizer object, PyTorch has the user explicitly call the adaptive loss scale
 update from a separate object.
 
-Refer to the [PyTorch AMP
-Recipe](https://pytorch.org/tutorials/recipes/recipes/amp_recipe.html) for a detailed walkthrough.
+
+We will use the 2022 release NVIDIA-optimiized NGC Singularity container for
+PyTorch 3.x in this walkthrough:
+```
+singularity exec --nv -B /lus /lus/theta-fs0/projects/datascience/zhen/docker/pytorch_22.01-py3.sif bash
```

Comment:
switch to a container in `/lus/theta-fs0/software/thetagpu/nvidia-containers/pytorch/`

Reply:
Can you help me to download the pytorch-22.01-py3 to this directory?